tests/test_crash_reports.py

```diff
@@ -29,11 +34,18 @@ def test_that_reports_form_has_same_product(self, mozwebqa, product):
         assert crash_adu.header.current_product == crash_adu.product_select
 
     @pytest.mark.nondestructive
-    @pytest.mark.parametrize(('product'), _expected_products)
+    @pytest.mark.parametrize(('product'), [
+        'Firefox',
+        pytest.mark.xfail(reason='bug 1233518')('Thunderbird'),
+        pytest.mark.xfail(reason='bug 1233518')('SeaMonkey'),
+        'FennecAndroid',
+        'WebappRuntime',
+        'B2G'])
     def test_that_current_version_selected_in_top_crashers_header(self, mozwebqa, product):
         csp = CrashStatsHomePage(mozwebqa)
         csp.header.select_product(product)
         cstc = csp.header.select_report('Top Crashers')
+        cstc.header.select_version('Current Versions')
```

Comment:
There was a behavior change, that said the test still seems to be needed in it's modified state.